IssueTracker/IssueDb/dbo/StoredProcedures/Participant/spParticipant_GetOwnersAndCollabsByProjectId.sql

```diff
@@ -5,6 +5,6 @@ begin
 	select *
 	from dbo.[Participant]
 	left join dbo.[User] ON [User].Id = [Participant].UserId AND [User].IsDeleted = 0
-	left join dbo.[Role] ON [Role].Id = [Participant].RoleId AND ( [Role].Name='Owner' OR [Role].Name = 'Collaborator')
-	where [Participant].IsDeleted = 0 AND [Participant].ProjectId = @ProjectId
+	left join dbo.[Role] ON [Role].Id = [Participant].RoleId 
+	where [Participant].IsDeleted = 0 AND [Participant].ProjectId = @ProjectId AND ([Participant].RoleId = 3 OR  [Participant].RoleId = 4)
```

Comment:
Add comment after the numbers (3,4) what represents each of them. Ex: 3 // Owner